extensions/amp-inputmask/0.1/constants.js

```diff
@@ -29,12 +29,7 @@ export const MaskChars = {
 export const MASK_SEPARATOR_CHAR = ' ';
 
 export const NamedMasks = {
-  EMAIL: 'email',
-  PHONE: 'phone',
-  PHONE_US: 'phone-us',
-  DATE_INTL: 'date-intl',
-  DATE_US: 'date-us',
-  DATE_ISO: 'date-iso',
+  CREDIT: 'credit',
```

Comment:
Why remove the other ones?

Reply:
They're not implemented yet, so I will re-add their definitions in the coming PRs that implement the named masked.